### ISSUE TITLE
Feature Request: Be able to select which devices to update in batch.

### ISSUE THREAD

**Issue by LewisSpring** [esphome/dashboard#141]
See [Discord Messages](https://discordapp.com/channels/429907082951524364/900481281958223903/911679082419331075)

Be able to select with tickboxes or similar which devices to update. This means that a user doesn't need to waste time updating devices that are offline.

If needed, the tickboxes allow for a user to attempt to update devices that the Dashboard thinks is offline.

**Comment by noxhirsch** [comment]
This would also allow you to easily update multiple devices with the same base configuration with substitutions if you made changes to the base configuration.

**Comment by lociii** [comment]
This would be so awesome. Running many ESPHome devices (around 50 in my case) is a pain to update when you don't want to update all at once (e.g. not the one that controls the computer and desk you're doing the update from).

**Comment by stleusc** [comment]
Yes please! Or detecting which devices failed and still need the update. Usually, one out of my devices will not update when there is a new version. Would be nice if the dashboard would detect that missing update and also then only update those missed nodes.

**Comment by dahasolutions** [comment]
This would be so awesome
